test/integration/auth/unauthenticated.test.js

```diff
@@ -29,4 +29,54 @@ describe('Query.customer without authorization header', () => {
     expect(result.data.customer).toBeNull()
     expect(result.errors).toHaveLength(1)
   })
+
+  it('business query should return an auth error', async () => {
+    const unAuthedContext = await context({})
+    const result = await graphql({
+      source: `#graphql
+        query Business {
+          business(sbi: "107183280") {
+            land {
+              parcels {
+                id
+                sheetId
+                area
+              }
+            }
+          }
+        }
+      `,
+      schema,
+      contextValue: unAuthedContext
+    })
+    expect(result.data.business).toBeNull()
+    expect(result.errors).toHaveLength(1)
```

Comment:
Does the error need to be unpacked to ensure the call is failing for the right reason? Same for following test too...

Reply:
good point, added